evm_arithmetization/src/cpu/kernel/aggregator.rs

```diff
@@ -9,10 +9,15 @@ use super::assembler::{assemble, Kernel};
 use crate::cpu::kernel::constants::evm_constants;
 use crate::cpu::kernel::parser::parse;
 
-pub const NUMBER_KERNEL_FILES: usize = if cfg!(feature = "cdk_erigon") {
-    159
+pub const NUMBER_KERNEL_FILES: usize = if cfg!(feature = "eth_mainnet") {
+    157
+} else if cfg!(feature = "cdk_erigon") {
+    156
+} else if cfg!(feature = "polygon_pos") {
+    155
 } else {
-    158
+    // unreachable
+    0
```

Comment:
You can't compile if you do not have a `usize` (like if you were to rely on `unreachable!()` for instance), and I didn't want to have the `else` only be equivalent to the last network feature in case we were to add another chain later on (as it would easily mess things up)